config/mail.php

```diff
@@ -57,7 +57,8 @@
 
     'from' => [
         'address' => env('MAIL_FROM_ADDRESS', 'hello@example.com'),
-        'name' => env('MAIL_FROM_NAME', 'Example'),
+        'name' => 'B-locker',
+        'subject' => 'Your locker',
```

Comment:
I wouldn't set 'Your locker' as default subject, or any default subject for that matter. A subject should be unique per email.